docs/Workflow.md

```diff
@@ -1,32 +1,55 @@
 <!--- SPDX-License-Identifier: Apache-2.0 -->
 # Contribution Guide
 
-## Prerequisite: Install LLVM
+## Step 1: Fork ONNX-MLIR in the cloud
 
-Follow directions in this section of the top level [README](../README.md#mlir) 
-for installing the currently supported LLVM version. We assume here a
-LINUX installation. 
+We strongly encourage contributors to work in their own forks of the ONNX-MLIR project.
+Creating a new fork is easy.
 
-### Step 1: Fork in the cloud
-
-1. Visit https://github.com/onnx/onnx-mlir
+1. Visit https://github.com/onnx/onnx-mlir.
 2. Click `Fork` button (top right) to establish a fork.
 
-### Step 2: Clone fork to local storage
+For the rest of this guide, we assume that you have set `user` to match your github profile name:
+
+```sh
+user={your github profile name}
+```
+
+## Step 2: Setup MLIR 
+
+### Setup using Docker
+
+Follow the directions provided [here](Docker.md#building-onnx-mlir-in-a-docker-environment) to establish a docker image that uses your ONNX-MLIR fork.
```

Comment:
Use the template provided in [here](Docker.md#building-onnx-mlir-in-a-docker-environment) to establish a docker image that uses your ONNX-MLIR fork by modifying it as follows:

1. Since the base image used by the template already contains a clone of the ONNX-MLIR main repository, in step 5, add your fork as a remote repository by uncommenting:
```
RUN git remote add origin https://github.com/<<GitID>>/onnx-mlir.git
```

As a best practice, uncomment the line which disables the pushing to upstream to avoid accidental pushes:
```
RUN git remote set-url --push upstream no_push
```

At the end of the two commands above:
- `upstream` will refer to the original ONNX-MLIR repository.
- `origin` will refer to your own fork of ONNX-MLIR.


2. Uncomment either step 3 or 4 depending on whether you plan to use VSCode for in conjunction with the ONNX-MLIR image.

3.  By default, ONNX-MLIR is built in `Debug` mode. Make the appropriate changes in step 6 if you wish to build ONNX-MLIR in `Release` mode.